src/main/java/Model/Winner.java

```diff
@@ -0,0 +1,26 @@
+package Model;
+
+import Model.Car;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winner {
+    private static final List<String> winnerList = new ArrayList<>();;
```

Comment:
```suggestion
    private final List<String> winnerList = new ArrayList<>();
```

위의 코드와 무엇이 다를지 고민해보시고, 어떤 것을 쓰는 것이 좋을지 판단하셔서 그 이유를 제게 알려주세요!

Reply:
단순히 한 게임을 한다고 생각해서 static으로 정의했는데 여러 게임이 있을 때는 winner 호출을 했을 때 문제가 생길 수 있기에 static을 빼줘야 할 것 같습니다.